features/facebookImport/rollup.config.js

```diff
@@ -51,13 +51,13 @@ export default (commandLineArgs) => {
                         dest: "dist/css",
                     },
                     {
-                        src: ["src/static/*", "!src/static/fonts"],
+                        src: ["src/static/"],
```

Comment:
Er. No. Eliminating symbolic links involves all kind of changes in the code, and I wouldn't want to go into that again